brave-jersey2/src/test/java/com/github/kristofa/brave/jersey2/ITBraveTracingFeature_Server.java

```diff
@@ -59,11 +52,33 @@ public Response disconnect() throws IOException {
     newServer(BraveTracingFeature.builder(brave).spanNameProvider(spanNameProvider).build());
   }
 
+  /**
+   * {@link ContainerResponseFilter} has no means to handle uncaught exceptions. Unless you provide
+   * a catch-all exception mapper, requests that result in unhandled exceptions will leak until they
+   * are eventually flushed.
+   */
+  @Provider
+  public static class ExceptionMapper implements ExtendedExceptionMapper<Exception> {
```

Comment:
Yea I wouldn't add this as a default as that would conflict with other registered mappers.